libs/template/themes/icons/mappers.ts

```diff
@@ -0,0 +1,26 @@
+import { IconMapper } from '@spryker-oryx/experience';
+import { IconTypes } from './icon.model';
+
+export const materialMapper: IconMapper = {
+  id: 'material-icons',
+  styles: `
+    font-family: 'Material Icons';
+    font-weight: normal;
+    font-style: normal;
+    font-size: 24px;
```

Comment:
```suggestion
    font-family: 'Material Symbols Outlined';
    font-variation-settings: "FILL" var(--icon-fill, 0), "wght" var(--icon-weight, 500), "GRAD" var(--icon-grad, 0), "opsz" var(--icon-optical, 48);
    font-size: var(--oryx-icon-size, 24px);
    align-self: center;
    -webkit-font-smoothing: antialiased;
```

We can most likely make it leaner when we drop the span inside the icon component, so we can leverage some of those rules from the host styles.